### PR TITLE
Add "Icons" extension

### PR DESCRIPTION
**Pull request by Destiner**
## Description

Adds a new extension to search, view, and copy icons from the [Iconify](https://iconify.design) icon sets.

## Screencast

https://user-images.githubusercontent.com/4247901/159638422-b172b8b3-76bf-4e43-98a0-9a3015b52eec.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/icons/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## Add Icons (2022-03-24)
+Initial version
```
Comment (FezVrasta) [inline extensions/icons/CHANGELOG.md:4]:
```suggestion
## [Add Icons] - 2022-03-24
- Initial version
```

https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history

File: extensions/icons/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "icons",
```
Comment (pernielsentikaer) [inline extensions/icons/package.json:3]: I would rename the extension and the folder to `iconify` 
```suggestion
  "name": "iconify",
```

File: extensions/icons/package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  "title": "Icons",
```
Comment (pernielsentikaer) [inline extensions/icons/package.json:4]:
```suggestion
  "title": "Iconify",
```

File: extensions/icons/package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  "license": "MIT",
+  "scripts": {
+    "build": "ray build -e dist",
+    "dev": "ray develop",
+    "lint": "ray lint"
+  },
+  "commands": [
+    {
+      "name": "view-icons",
+      "title": "View Icons",
```
Comment (pernielsentikaer) [inline extensions/icons/package.json:21]:
```suggestion
      "title": "Search Icons",
      "subtitle": "Iconify",
```

Comment (pernielsentikaer) [reply extensions/icons/package.json:21]: Could be View icons still though

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Destiner 👋 

Thanks for another awesome contribution 🔥

I left some minor feedback for you to address, could you look into this?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by Destiner** [comment]
Hey @pernielsentikaer, thanks for review! I updated the command title.

As for changing the extension title, I'd like to keep the current one if possible. While it's true that the extension uses Iconify, I think that it's not relevant for the end users. Most people are probably not aware of Iconify, but they will still find the extension useful.

**Comment by pernielsentikaer** [comment]
Hi @Destiner 👋 

Thanks for your update, much appreciated 🙏 

The name `Icons` is just really not saying anything, according to the [Naming convention](https://developers.raycast.com/basics/prepare-an-extension-for-store#extensions-and-commands-naming) we have to make it a bit more specific, maybe something like `Search Iconify Icons` could be an idea?

**Comment by Destiner** [comment]
@pernielsentikaer how about "Icon Search" or "Icon Sets"?

**Comment by pernielsentikaer** [comment]
Hi @Destiner 👋 

In my opinion it's still too generic, we already have `CSS.GG` and `Copy GCP Icons`, so `Icon Search` and `Icon Sets` doesn't really say much about what and which service it uses – Could we settle on something like `Iconify Search` and then use the description a bit more?

Catch me in [Slack](https://raycastcommunity.slack.com/) or here if we need to discuss further 😄

**Comment by Destiner** [comment]
@pernielsentikaer I changed the extension name to `Iconify — Search Icons`.

**Review by pernielsentikaer — APPROVED** [review]
Hi @Destiner 👋 

Thanks for taking your time, and sorry for pushing you for the name 😄 

Lets get it launched! 🚀

**Comment by apithy-isidro-martinez** [comment]
could you add the [heroicons](https://heroicons.com/) icons sets from [TailwindCSS Team](https://tailwindcss.com/) ? thanks

**Pull request merged**